app/src/androidTest/java/com/pileproject/drive/execution/BlockProgramLogicTest.java

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
There's classic golden rule for unit testing: [Arrange-Act-Assert](http://defragdev.com/blog/?p=783).
I'd like to recommend that you should follow this.